src/index.js

```diff
@@ -1,11 +1,23 @@
 var bodyParser = require('body-parser');
 var app = require('express')();
 var handleWfhRequest = require('./handle-wfh-request');
+var parseText = require('./parse-text')
 
 app.use(bodyParser.urlencoded({ extended: true }));
 app.listen(process.env.PORT);
 
 app.post('/wfh', (req, res) => {
 	res.send();
-	handleWfhRequest(req.body.user_id, req.body.response_url);
+	if (parseText.checkIfClear(req.body.text))
```

Comment:
Historically, this was written to be a simple command which took no arguments, thus no explicit "clear" command. Now that we take arguments, I sorta see the merits of both arguments. @TNagaraja I like that you're thinking about how changing the toggle "breaks the public API" of this command, but @MiguelCastillo also makes a good point about making the user aware of a "destructive" action. In the spirit of code reviews, @TNagaraja, I'm going to let you make the final decision. Thankfully, since this is an internal tool, we only have a small user base that will need to be informed about any changes to this command.